### PR TITLE
Implementation of issue #336

### PR DESCRIPTION
**Pull request by oliverschaefer**
Added `hasBeenPresented` method to address issue #336

### CODE REVIEW

File: ResearchKit/Common/ORKStepViewController.h [hunk 1]
```diff
@@ -350,6 +350,13 @@ ORK_CLASS_AVAILABLE
  */
 - (void)goBackward;
 
+/**
+ Returns a Boolean value indicating whether the step has been presented before.
+ 
+ @return `YES` if the step has been presented; otherwise `NO`.
+ */
+- (BOOL)hasBeenPresented;
```
Comment (YuanZhu-apple) [inline ResearchKit/Common/ORKStepViewController.h]: Can you use a property with `readonly` attribute, which is a more prefered way to return a Boolean in RK.

File: ResearchKit/Common/ORKStepViewController.h [hunk 2]
```diff
@@ -350,6 +350,13 @@ ORK_CLASS_AVAILABLE
  */
 - (void)goBackward;
 
+/**
+ Returns a Boolean value indicating whether the step has been presented before.
```
Comment (YuanZhu-apple) [inline ResearchKit/Common/ORKStepViewController.h]: `the view controller` has been presented

**Comment by oliverschaefer** [comment]
@YuanZhu-apple Done.

**Comment by YuanZhu-apple** [comment]
r=me.

**Comment by umerkhan-apple** [comment]
r=m3

**Pull request merged**